deployments/charts/kuma/templates/cni-daemonset.yaml

```diff
@@ -51,13 +51,30 @@ spec:
       containers:
         - name: install-cni
           {{- if .Values.experimental.cni }}
-          image: {{ include "kuma.formatImage" (dict "image" .Values.cni.imageExperimental "root" $) | quote }}
+          image: {{ include "kuma.formatImage" (dict "image" .Values.cni.experimental.image "root" $) | quote }}
           imagePullPolicy: IfNotPresent
+          readinessProbe:
+            initialDelaySeconds: {{ .Values.cni.test.sleepBeforeRunSeconds }}
+            exec:
+              command:
+                - cat
+                - /tmp/ready
+            {{- if .Values.cni.test.sleepBeforeRunSeconds }}
+          command: [ "/bin/sh", "-c", "--" ]
+          args: [ "sleep {{.Values.cni.test.sleepBeforeRunSeconds}} && exec /install-cni" ]
```

Comment:
A sleep 0 is not sleep so maybe it's simpler to not have a if?

Reply:
this is inside an if `{{- if .Values.cni.test.sleepBeforeRunSeconds }}` so this will only be applied if `sleep > 0`